code/ui/manager/src/runtime.ts

```diff
@@ -55,12 +58,40 @@ class ReactProvider extends Provider {
   }
 }
 
-const { document } = global;
-
-const rootEl = document.getElementById('root');
-renderStorybookUI(rootEl, new ReactProvider());
-
 // Apply all the globals
 Object.keys(Keys).forEach((key: keyof typeof Keys) => {
   global[Keys[key]] = values[key];
 });
+
+globalThis.sendTelemetryError = (error) => {
+  const channel = globalThis.__STORYBOOK_ADDONS_CHANNEL__;
+  channel.emit(TELEMETRY_ERROR, error);
+};
+
+function preprocessError(originalError: Error) {
```

Comment:
Do we need to do this in another places?

Reply:
no, basically just in `manager` because we assume that every error that occurs there is related to Storybook, but as we are catching all errors (instead of only the ones with error.fromStorybook), we should be mindful of many error shapes which involve circular references to the window object